### PR TITLE
Add Nyte

### PR DESCRIPTION
**Pull request by SLACKIEY**
adding nyte.cc for lunar server mappings

### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

**Review by GrowlyX — CHANGES_REQUESTED** [review]
Please remove your `.idea` files from this pull request.

**Comment by SLACKIEY** [comment]
> Please remove your `.idea` files from this pull request.

done

File: servers.json [hunk 1]
```diff
@@ -1524,3 +1531,4 @@
         ]
     }
 ]
+
```
Comment (GrowlyX) [inline servers.json:1534]: Please remove this blank line. Also, the file name of your logo must match this format `<id>.png`. You currently have `nytecc.png`, it must be `nyte.png`.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (SLACKIEY) [reply servers.json]: done

**Review by SLACKIEY — COMMENTED** [review]
Fixed all that needed fixed

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please add the following fields to your servers.json:

primaryAddress
minecraftVersions
primaryMinecraftVersion

Values for these can be found in the README

**Comment by SLACKIEY** [comment]
Done

File: servers.json [hunk 2]
```diff
@@ -1438,6 +1438,16 @@
...
         ]
     },
+    {
+        "id": "Nyte",
+        "name": "Nyte",
+        "addresses": [
+            "nyte.cc"
+        ],
+        "primaryAddress": "nyte.cc",
+        "minecraftVersions": ["1.8.9", "1.19"],
```
Comment (itsJordan) [inline servers.json:1448]:
```suggestion
        "minecraftVersions": ["1.8", "1.19"],
```

File: servers.json [hunk 3]
```diff
@@ -1438,6 +1438,16 @@
...
+        "primaryMinecraftVersion": "1.8.9"
```
Comment (itsJordan) [inline servers.json:1449]:
```suggestion
        "primaryMinecraftVersion": "1.8"
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please change the following:

- Adjust your logo to be 512x512 pixels.
- Remove the minor update version from your minecraftVersions + primaryMinecraftVersion fields (suggested)
- Add all supported minecraft versions from this list: "1.7", "1.8", "1.12", "1.16", "1.17", "1.18", "1.19"

File: servers.json [hunk 4]
```diff
@@ -1438,6 +1438,16 @@
             "antique.gg"
         ]
     },
+    {
+        "id": "Nyte",
```
Comment (itsJordan) [inline servers.json:1442]:
```suggestion
        "id": "nyte",
```

Comment (SLACKIEY) [reply servers.json:1442]: done

**Review by itsJordan — APPROVED** [review]

**Pull request merged**